contributions.html

```diff
@@ -129,23 +128,23 @@ <h1 id="history">History</h1>
 <div class="cpw pt mb">
 
    <ul>
-   <li>Steve Howard, <a href="http://paulbunyan.net" target="_blank">Paul Bunyan Telephone</a>, and Jay
-Hanke, <a href="http://mankatonetworks.com" target="_blank">Mankato Networks</a> talked about workarounds
-to the “Chicago Problem”</li>
+   <li>Steve Howard, <a href="https://paulbunyan.net/" target="_blank">Paul Bunyan Telephone</a>, and Jay
+Hanke, <a href="https://www.mankatonetworks.com/" target="_blank">Mankato Networks</a> talked about workarounds
+to the "Chicago Problem"</li>
```

Comment:
What's wrong with the smart quotes? This is UTF-8, right?

Reply:
I will revert that line and yes UTF-8 can handle high bits.